bin/node/runtime/src/lib.rs

```diff
@@ -513,6 +513,7 @@ parameter_types! {
 		.get(DispatchClass::Normal)
 		.max_extrinsic.expect("Normal extrinsics have a weight limit configured; qed")
 		.saturating_sub(BlockExecutionWeight::get());
```

Comment:
sidenote: I think we should have to substract base block weight here: https://github.com/paritytech/substrate/issues/8434